config/default.json

```
@@ -1,6 +1,8 @@
 {
-  "serverUrl": "http://localhost:8080",
-  "cdnUrl": "http://localhost",
+  "serverUrl": "random.snode",
+  "cdnUrl": "random.snode",
+  "messageServerPort": ":8080",
```

```suggestion
  "messageServerPort": "8080",
```
The `:` should be added in when building the url instead of have it in this property

---

Yep, having the `:` in port is weird.
You can do `port = port? `:${port}` : '';` if you want to allow the absence of port.

---

Tru, nice